test/e2e/gitops_service_test.go

```diff
@@ -301,3 +303,38 @@ func validateMachineConfigUpdates(t *testing.T) {
 	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: existingImage.Name}, existingImage)
 	assertNoError(t, err)
 }
+
+func validateClusterConfigChange(t *testing.T) {
+	framework.AddToFrameworkScheme(corev1.AddToScheme, &corev1.ConfigMap{})
+	ctx := framework.NewContext(t)
+	defer ctx.Cleanup()
+	f := framework.Global
+
+	schedulerYAML := filepath.Join("test", "yamls", "scheduler_appcr.yaml")
+	ocPath, err := exec.LookPath("oc")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	cmd := exec.Command(ocPath, "apply", "-f", schedulerYAML)
+	err = cmd.Run()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	time.Sleep(5 * time.Second)
```

Comment:
We probably don't want to add more `time.Sleep(...)` calls if we can help it. See https://github.com/redhat-developer/gitops-operator/issues/130 for outline of alternatives. 

You should be able to do something like this (pseudocode):
```go

// import this:
//	"k8s.io/apimachinery/pkg/util/wait"


err := wait.Poll(time.Second*1, time.Second*60, func() (bool, error) {
	
	if helper.ApplicationHealthStatus("scheduler", "openshift-gitops"); err != nil {
		// print the error received
		return false, nil
	}

	if helper.ApplicationSyncStatus("scheduler", "openshift-gitops"); err != nil {
		// print the error received	
		return false, nil
	}
	
	existingConfigMap := &corev1.ConfigMap{}
	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: "policy-configmap", Namespace: "openshift-config"}, existingConfigMap)
	if err != nil {
			// print the error received
			return false, nil
	}

		return true, nil
})
```

Reply:
I thought Amit was going to handle that in a separate task. But I can surely update this test.